ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -1062,3 +1068,171 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
 }
 
 @end
+
+#pragma mark - ORKFormItemLocationCell
+
+@interface ORKFormItemLocationCell () <ORKLocationSelectionViewDelegate>
+
+@property (nonatomic, assign) BOOL editingHighlight;
+@property (nonatomic, strong) NSString *placeholder;
+
+@end
+
+@implementation ORKFormItemLocationCell {
+    ORKLocationSelectionView *_selectionView;
+}
+
+- (void)cellInit {
+    [super cellInit];
+    
+    _selectionView = [[ORKLocationSelectionView alloc] init];
+    _selectionView.delegate = self;
+    
+    _selectionView.mapView.layer.cornerRadius = 3;
+    _selectionView.mapView.layer.borderColor = [UIColor colorWithRed:(204.0/255.0) green:(204.0/255.0) blue:(204.0/255.0) alpha:.75].CGColor;
+    _selectionView.mapView.layer.borderWidth = 1.0 / [UIScreen mainScreen].scale;
+    
+    if (_placeholder != nil) {
+        [_selectionView setPlaceholderText:_placeholder];
+    }
+    
+    [self applyAnswerFormat];
+    [self answerDidChange];
+    
+    [self.contentView addSubview:_selectionView];
+    
+    {
+        NSDictionary *dictionary = @{@"_selectionView":_selectionView};
+        ORKEnableAutoLayoutForViews([dictionary allValues]);
+        NSDictionary *metrics = @{@"vMargin":@(10), @"hMargin":@(self.separatorInset.left)};
+        
```

Comment:
The `vMargin` and `hMargin` constants have been refactored to `verticalMargin` and `horizontalMargin` in the last constraint code refactor merge.